app/models/host.rb

```diff
@@ -796,10 +796,8 @@ def verify_credentials_with_ssh(auth_type = nil, options = {})
       # connect_ssh logs address and user name(s) being used to make connection
       _log.info("Verifying Host SSH credentials for [#{name}]")
       connect_ssh(options) { |ssu| ssu.exec("uname -a") }
-    rescue MiqException::MiqInvalidCredentialsError
-      raise MiqException::MiqInvalidCredentialsError, _("Login failed due to a bad username or password.")
-    rescue MiqException::MiqSshUtilHostKeyMismatch
-      raise # Re-raise the error so the UI can prompt the user to allow the keys to be reset.
+    rescue Net::SSH::AuthenticationFailed
+      raise _("Login failed due to a bad username or password.")
```

Comment:
We may have to do the miqexception wrapping here to allow for the callers to handle it properly

Reply:
Oh I see it discussed below in https://github.com/ManageIQ/manageiq/pull/19843/files#r380765127